src/settings/validateNameAndCode.test.js

```diff
@@ -0,0 +1,19 @@
+import '../../test/jest/__mock__';
+import validateNameAndCode from './validateNameAndCode';
+
+describe('ValidateNameAndCode', () => {
+  it('should return errors when code & name are empty', () => {
+    const item = {};
+    const errors = validateNameAndCode(item);
+    expect(JSON.stringify(errors.code)).toMatch(/ui-inventory.fillIn/i);
+    expect(JSON.stringify(errors.name)).toMatch(/ui-inventory.fillIn/i);
+  });
+  it('should return empty with name & code values present', () => {
+    const item = {
+      code: 'codeTest',
+      name: 'codeTest'
+    };
+    const errors = validateNameAndCode(item);
+    expect(errors).toBeDefined();
```

Comment:
The stated purpose, "return empty with name & code values present" sounds great, but this test checks whether the return value is defined, not whether it is empty. Indeed, it needs to check that the returned object is empty in order to satisfy the question of whether errors are present.

Reply:
Test name is changed to 'check return value is defined'